ArcPicker/ArcPicker.js

```diff
@@ -0,0 +1,174 @@
+import kind from '@enact/core/kind';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import Arc from '../Arc';
+import ArcPickerBehaviorDecorator from './ArcPickerBehaviourDecorator';
+import Skinnable from '../Skinnable';
+
+import css from './ArcPicker.module.less';
+
+/**
+ * An Agate component for displaying fan speed {@link agate/ArcPicker}.
+ *
+ * @class ArcPickerBase
+ * @memberof agate/ArcPicker
+ * @ui
+ * @private
+ */
+const ArcPickerBase = kind({
+	name: 'ArcPickerBase',
+
+	propTypes: /** @lends agate/ArcPicker.ArcPickerBase.prototype */ {
+		/**
+		 * The value options of ArcPicker.
+		 *
+		 * @type {Array}
+		 * @public
+		 */
+		options: PropTypes.array.isRequired,
+
+		/**
+		 * The end angle(in degrees) of the arc.
+		 *
+		 * The value should be between 0 and 360 and should be greater than startAngle.
+		 *
+		 * @type {number}
+		 * @default 310
+		 * @public
+		 */
+		endAngle: PropTypes.number,
+
+		/**
+		 * Called when the path area is clicked.
+		 *
+		 * @type {Function}
+		 * @param {Object} event
+		 * @public
+		 */
+		onClick: PropTypes.func,
```

Comment:
Is there any way to deselect all arcs when selectionType is equal to `cumulative`?
I mean.. If the fan speed could be 0, 1, 2, 3, 4, 5, and 6, how could we make a value 0?
One of the possible solution is..
when clicking out of any Arcs, then we could make the value 0.
What do you think?

Reply:
I thought about this as well. I think it's a good approach. Fixed it.